api/civo/main.go

```diff
@@ -50,15 +50,29 @@ func Credentials() bool {
 }
 
 // fetchAPIKey returns the api_token from the cred/civo.json file store
-func fetchAPIKey() string {
+func fetchAPIKey() (string, error) {
 
 	token, err := util.GetCred("civo")
 
 	if err != nil {
-		return ""
+		return "", err
 	}
 
-	return token["token"]
+	return token["token"], err
+}
+
+// Authentication
+func authenticate() (string, error) {
+	apikey, err := fetchAPIKey()
+	if err != nil {
+		return "", err
+	}
+	civoToken := os.Getenv("CIVO_TOKEN")
+	if civoToken != "" {
+		return civoToken, nil
+	}
+	return apikey, nil
+
```

Comment:
make the CIVO_TOKEN Env check before the fetchAPIKey() also edit the fetchAPIKey no need to make it seperate

@AvineshTripathi should we have 2 seperate function one for env and another which access the token from the `cred.json`?

Reply:
I think we should support only one

Reply:
Yes